src/porepy/numerics/parabolic.py

```diff
@@ -5,7 +5,7 @@
 from porepy.params import tensor, bc
 from porepy.fracs import meshing
 from porepy.numerics.fv import fvutils, tpfa, mass_matrix, source
-from porepy.numerics.fv.transport import upwind, upwind_coupling
+from porepy.numerics.fv.transport import upwind
```

Comment:
This line really shows why it is correct to merge single and multi-domain solvers. From the outside, there should be no difference between them.